samples/fabric/simpleassettransfer/bondasset.go

```diff
@@ -364,19 +363,12 @@ func (s *SmartContract) GetAssetClaimStatus(ctx contractapi.TransactionContextIn
 		return blankClaimBytes64, err
 	}
 	if claimAssetDetails == nil {
-		return blankClaimBytes64, err
+		// represents the scenario that the asset was not claimed by the remote network
+		return blankClaimBytes64, nil
 	}
 
 	// Validate returned asset details using app-specific-logic
-
-	// The asset should be recorded if it has been claimed, so we should look that up first
-	asset, err := getBondAsset(ctx, assetType, id)
```

Comment:
I suppose now this function doesn't need `assetType` and `id` as parameters, can you make a note of it anywhere as TODO, to remove them?